src/components/Tokens/TokenDetails/About.tsx

```diff
@@ -106,7 +106,7 @@ export function AboutSection({ address, chainId, description, homepageUrl, twitt
       <ResourcesContainer data-cy="resources-container">
         <Resource
           name={chainId === ChainId.MAINNET ? 'Etherscan' : 'Block Explorer'}
-          link={`${explorer}${address === 'NATIVE' ? '' : 'address/' + address}`}
+          link={`${explorer}${address === 'NATIVE' ? '' : 'token/' + address}`}
```

Comment:
this would break for some explorers which don't use `token/` pathing. Can you refactor to use the helper function `getExplorerLink` instead?

Reply:
⬆️